app/Repositories/Admin/Project/ProjectRepository.php

```diff
@@ -195,4 +200,19 @@ public function togglePublicStatus($project): string
         }
         return 'Public Status toggled Successfully!';
     }
+
+    /**
+     * @param $projects
+     * @param $flag
+     * @return string
+     * @throws GeneralException
+     */
+    public function toggleStarter($projects, $flag): string
+    {
+        if (empty($projects)) {
+            throw new GeneralException('Choose at-least one project.');
+        }
+        $this->model->whereIn('id', $projects)->update(['starter_project' => (bool)$flag]);
+        return 'Projects Starter status Updated Successfully!';
```

Comment:
Updated Successfully does not need to be upper case.

Reply:
Done